timesketch/lib/utils.py

```diff
@@ -176,7 +186,8 @@ def read_and_validate_csv(
                     'Rows {0} to {1} skipped due to malformed '
```

Comment:
If the warning string is only used once, it could also be formatted directly in the ```logger.warning()``` call, WDYT?

Reply:
I agree, however I was trying to keep the style consistent with a similar warning message a few lines below.